projects/sw-appcache-behavior/test/automated-suite.js

```diff
@@ -38,6 +38,19 @@ describe('sw-appcache-behavior Test Suite', function() {
   this.timeout(TIMEOUT);
   let baseTestUrl;
 
+  // Helper method to generate the text of an App Cache manifest with a
+  // specific version.
+  const generateManifestText = version => {
+    return `CACHE MANIFEST
+# Version: ${version}
+
+CACHE:
+../common.css
+
+NETWORK:
+*`;
+  };
+
   // Set up the web server before running any tests in this suite.
   before(function() {
     return testServer.startServer('.').then(portNumber => {
```

Comment:
You might want to download browsers if missing + the firefox driver like so: https://github.com/web-push-libs/web-push/blob/master/test/helpers/download-test-browsers.js